keras_rs/src/layers/modeling/dot_interaction.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any, List, Tuple
```

Comment:
Please use `list` instead of `List` and `tuple` instead of `Tuple` for type annotations for consistency. So you only need to import `Any`.